src/CashuWallet.ts

```diff
@@ -266,7 +336,7 @@ class CashuWallet {
 		const blindedMessages: Array<SerializedBlindedMessage> = [];
 		const secrets: Array<Uint8Array> = [];
 		const rs: Array<bigint> = [];
-		const count = Math.ceil(Math.log2(feeReserve));
+		const count = Math.ceil(Math.log2(feeReserve)) || 1;
```

Comment:
should we back propagate this change to the other implementations? Or are they already aware of this

Reply:
not sure about this

```typescript
Math.ceil(Math.log2(1)) => 0
```

can the LN fee be 1?
If it can and we pay 0 sats fee, the we won't get a change back with `Math.ceil(Math.log2(1))` but we a should get 1 sat back or I'm missing something?

Reply:
the change is good👍 i was just wondering how other implementations handle it, and if we should let them know